src/layouts/Header/ControlBar/ControlBar.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function ControlBar() {
```

Comment:
name of this component isn't obvious

Reply:
Removed this component. He is redundant here